src/analysis/patternSerializer.ts

```diff
@@ -18,7 +17,9 @@ const KNOWN_FUNCTIONS = {
 
 export function loadPattern(patternString: string): RoutePatternDefinition {
   const routeSplit = patternString.split("||");
-  assert(routeSplit.length === 2);
+  if (routeSplit.length !== 2) {
+    throw "routeSplit.length !== 2";
+  }
```

Comment:
why did you change the assertions into throws?

Reply:
Because `assert` is not a native module and I don't wanna use `console.assert` because we have not decided our final environment. You can use `npm run build` to see the error it throws.

Reply:
maybe let's install `assert` as a dependency?

Reply:
Yeah I have installed it now.

Reply:
`assert` is a CommonJS module, so it does not really fit in automatically. I have switched to `console.assert` for now. Let me know if you have a better idea.